back-office/international/translations/translations.md

```diff
@@ -1,12 +1,12 @@
 # Translations
 
-## Conceptual vocabulary
+#### Conceptual vocabulary
 
-- Wordings: original texts found throughout the software. They are in English but may be translated _to_ English for better localization.
-- Translations: wordings translated to a given language.
-- Translation domain: named logic group of wordings. Any given wording is attached to one and only one translation domain. Two identical wordings attached to two different translation domains are considered as two different wordings.
+- **Wordings:** original texts found throughout the software. They are in English but may be translated _to_ English for better localization.
+- **Translations:** wordings translated to a given language.
+- **Translation domain:** named logic group of wordings. Any given wording is attached to one and only one translation domain. Two identical wordings attached to two different translation domains are considered as two different wordings.
 - Translation catalogue: A collection of translation domains containing translations for a given language, stored physically in a support like files or on a database.
-- Default catalogue: The master translation catalogue that contains the wordings but not its translations. It's [included in the GitHub project](https://github.com/PrestaShop/PrestaShop/tree/1.7.6.0/app/Resources/translations/default).
+- **Default catalogue:** The master translation catalogue that contains the wordings but not its translations. It's [included in the GitHub project](https://github.com/PrestaShop/PrestaShop/tree/1.7.6.0/app/Resources/translations/default).
```

Comment:
```suggestion
- **Default catalog:** The master translation catalog that contains the wordings but not its translations. It's [included in the GitHub project](https://github.com/PrestaShop/PrestaShop/tree/1.7.6.0/app/Resources/translations/default).
```

Reply:
We use "catalogue" (alternative spelling) because that's the one chosen by Symfony's implementation.